airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/request_options/request_options_provider.py

```diff
@@ -20,7 +20,13 @@ class RequestOptionsProvider(ABC):
     """
 
     @abstractmethod
-    def request_params(self, **kwargs) -> MutableMapping[str, Any]:
+    def request_params(
```

Comment:
fixed interface shared by all subclasses and force the parameters to be named to avoid messing up the order